lib/common/MongoDbLogger.ts

```diff
@@ -0,0 +1,31 @@
+import { LoggerState, MongoClient } from 'mongodb';
+import Logger from '../common/Logger';
+
+/**
+ * Base class that contains the common logging functions for mongodb.
+ */
+export default class MongoDbLogger {
```

Comment:
Design: this class probably won't be necessary when you are done with `MongoDbStore` refactoring as you can probably hide these methods within it.

Reply:
Still prefer to remove this class and just have the static methods in `MongoDbStore` as it only makes sense in that context. Happy to discuss.